python/ray/autoscaler/autoscaler.py

```diff
@@ -64,6 +88,101 @@
 # Max number of nodes to launch at a time.
 MAX_CONCURRENT_LAUNCHES = 10
 
+# Interval at which to perform autoscaling updates.
+UPDATE_INTERVAL_S = 5
+
+# The autoscaler will attempt to restart Ray on nodes it hasn't heard from
+# in more than this interval.
+HEARTBEAT_TIMEOUT_S = 30
```

Comment:
I still think it'd be a good idea to have a separate file with all of the constants. That way people don't have to search through all the files to find all the constants they need to change while debugging. What do you think about putting these in a separate `ray_constants.py` file?

Reply:
Done